src/kbmod/analysis/visualization_utils.py

```diff
@@ -0,0 +1,18 @@
+from astropy.time import Time
+
+
+def mjd_to_day(mjd):
+    """Takes an mjd and converts it into a day in
+    calendar date format.
+
+    Parameters
+    ----------
+    mjd : `float`
+        mjd format date.
+
+    Returns
+    ----------
+    A `str` with a calendar date, in the format YYYY-MM-DD.
+    e.g., mjd=60000 -> '2023-02-25'
+    """
+    return str(Time(mjd, format="mjd").to_value("datetime")).split()[0]
```

Comment:
It should be possible to print a timestamp in whatever format you want, including custom C-style formatting, without making an entire module for it. 

https://docs.astropy.org/en/stable/api/astropy.time.Time.html

```
>>> from astropy.time import Time
>>> t = Time(56000, format="mjd")
>>> t.fits
'2012-03-14T00:00:00.000'
>>> t.yday
'2012:074:00:00:00.000'
>>> t.ymdhms
(2012, 3, 14, 0, 0, 0.)
>>> t.iso
'2012-03-14 00:00:00.000'
>>> t.jyear
2012.1984941820672
>>> t.iso
'2012-03-14 00:00:00.000'
>>> t.strftime("%Y-%M-%d")
'2012-00-14'
>>> t.to_datetime()
datetime.datetime(2012, 3, 14, 0, 0)
```

Reply:
I'm planning on expanding the `visualization_utils` module with other functions, but for the first PR I just wanted to have everything set up to replicate the TNO notebook functionality

Reply:
I think it is okay to leave in for now given how it is used in the other file (to generate a key for a hash table). But if we don't end up adding more to `visualization_utils` we should clean it up.

Reply:
Can we at least avoid casting to two different objects and making a copy of one with:

> t.strftime(fmt)

Reply:
Sounds good. Let's do that.